### DISCUSSION TITLE
Absolute `line number' on focus lose in a terminal

### DISCUSSION THREAD

**Discussion by woojiq** [helix-editor/helix#7954] [Ideas] [answered]
Currently, if you have more than one window inside the helix and have "relative" line numbers, it automatically converts to absolute line numbers in inactive windows.

> Line number display: absolute simply shows each line's number, while relative shows the distance from the current line. When unfocused or in insert mode, relative will still show absolute line numbers

However, if you split terminal (not the helix), lines will remain relative. I'd like to see them as absolute too (because actually that window is inactive), it's useful for running git blame (or something similar) rather than calculating the start/end of a line in my head.
I have implemented this locally, if this is the desired behavior I can create a pr (small).
Is there any reason why this wasn't added earlier?

**Comment by gabydd** [comment, marked as answer]
I'd say that counts as unfocused if you are in a different terminal split, there probably isn't a reason this wasn't implemented so it should be fine to make a pr we can review the implementation there
